src/OV/OpenVarioMenu.cpp

```diff
@@ -467,6 +468,9 @@ Main()
 
 int main()
 {
+  /*the x-menu is waiting a second to solve timing problem with display rotation */
+  std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
This sounds really horrible, and the justification for such a horrible kludge must be documented really well. Just saying "timing problem" isn't enough.